client/src/modules/debtors/groups.update.html

```diff
@@ -23,9 +23,19 @@
 
           <div
             class="form-group has-feedback"
-            ng-class="{'has-error' : debtorGroup.color.$invalid && debtorGroup.$submitted}">
+            ng-class="{'has-error' : !GroupUpdateCtrl.group.color && debtorGroup.$submitted}">
             <label class="control-label" translate>FORM.LABELS.COLOR</label>
-            <input name="color" id="title-color" type="color" ng-model="GroupUpdateCtrl.group.color" class="form-control" ng-value="GroupUpdateCtrl.group.color">
+
+            <ui-select tagging ng-model="GroupUpdateCtrl.group.color" theme="bootstrap">
```

Comment:
You don't need `theme="bootstrap"` here.  We already define it in our `app.js` configuration.  See [this file](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/js/app.js#L176).